SeriesMaster-Frontend/App/components/CardImage.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "styled-components/native";
+import PropTypes from "prop-types";
+
+const StyledCardImage = styled.Image`
+  height: 125px;
+  width: 125px;
+  border-radius: 25px;
+  margin: 5px;
+`;
+
+const CardImage = ({ image }) => {
+  return <StyledCardImage source={image} />;
+};
+
+export default CardImage;
+
+CardImage.propTypes = {
+  image: PropTypes.any,
```

Comment:
I would change the name to clarify, that this is the path to the source. 

```suggestion
  imageSrc: PropTypes.string,
```